system_tests/logging_.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+import unittest2
+
+from gcloud import _helpers
+from gcloud.environment_vars import TESTS_PROJECT
+from gcloud import logging
+
+
+DEFAULT_LOGGER_NAME = 'system-tests-%d' % (1000 * time.time(),)
+
+
+class Config(object):
+    """Run-time configuration to be modified at set-up.
+
+    This is a mutable stand-in to allow test set-up to modify
+    global state.
+    """
+    CLIENT = None
+
+
+def setUpModule():
+    _helpers.PROJECT = TESTS_PROJECT
```

Comment:
I don't know why this has never occurred to me before. Should we restore the original value in `tearDownModule`?